app/ts/Workspace/WorkspaceContext.tsx

```diff
@@ -86,23 +111,153 @@ export function WorkspaceContextProviderComponent({
       .catch(errorCallback);
   }
 
+  function editTherapeuticContext(therapeuticContext: string): void {
+    const oldWorkspaceToSend: IOldWorkspace = _.merge(
```

Comment:
We can just use spread operator for this now I think instead of _.merge

Reply:
Using spreading is more complex due to the property being changed being several steps into the object.